### ISSUE TITLE
docker-compose support

### ISSUE THREAD

**Issue by grigio** [Xtrendence/Cryptofolio#47]
I think it's important to separate cryptofolio code from userdata
I tried to mount `/var/www/html/api/data` as volume but it seems the initial config files aren't generated

```
A /var/www/html/api/data/account.json
A /var/www/html/api/data/activity.json
A /var/www/html/api/data/coins.json
A /var/www/html/api/data/holdings.json
A /var/www/html/api/data/settings.json

```

**Comment by Xtrendence** [comment]
Thank you, I'll look into this. The initial files are generated by the API so that it's clear there's a permission or location issue if they aren't, rather than letting the user login and then have the rest of the application not work right. In the case of docker-compose, I'd have to experiment and see how that would work, because I only started using Docker with this project, so I only know the very basics.

**Comment by Xtrendence** [comment]
Fixed in commit 4f6ee72eab2edec39578d132c68e0c5f5043e945, which adds support for docker-compose.
